codecs/imagequant/build.sh

```diff
@@ -14,6 +14,7 @@ echo "============================================="
   emcc \
     --bind \
     ${OPTIMIZE} \
+    --closure 1 \
```

Comment:
We don’t need this here as we are just generating object files

Reply:
Oh right, there's a `-c` there... Fixed and I removed some other unnecessary flags as well.